src/basic-syntax/compound-types.md

```diff
@@ -24,3 +24,33 @@ fn main() {
     println!("2nd index: {}", t.1);
 }
 ```
+
+<details>
+    
+Key points:
+    
+Arrays:
+    
+*Arrays have elements of the same type, T, and length, N, which is fixed. 
+
+*We can use literals to assign values to arrays.
+
+*Demonstrate out of bounds errors by setting n to different values 
+    e.g. a[n+15] = 11 // index out of bounds error since len is 10
+
+*Efficient way to check n is in bounds:
+    assert!(n+20 < a.len()); // panics
```

Comment:
The newline here and elsewhere is not significant in Markdown.

Reply:
Ack of course. Apologies! Updated PR